spec/components/schemas/Payments/ResponseSources/PaymentResponseSofortSource.yaml

```diff
@@ -0,0 +1,20 @@
+###
+# Warning: this file was auto generated from OpenAPI specs. Do not manually edit. 
```

Comment:
why the Sofort-related schemas were added?

Reply:
@alex-larin-cko , the script adds all the schemas specified in the [sync-generated-apm-specs.js](https://github.com/cko-apm/checkout-api-reference/pull/6/files#diff-055515cc9dc6be90e6e04185db895ad0fa4dbe32c485dbdc74104e7b9e35f0ca). For some reasons the `Sofort` schemas were not present. But it should not be an issue, since the `spec` folder does not affect the documentation. So, could we leave it as it is?

Reply:
sure, just asking. Anyway, we will need to deploy this and review it with Products for final approval.